SDK/simQt.h

```diff
@@ -40,7 +40,7 @@
 #include "simQt/DataTableComboBox.h"
 #include "simQt/DataTableModel.h"
 #include "simQt/DirectorySelectorWidget.h"
-#include "simQt/DndTreeView.h
+#include "simQt/DndTreeView.h"
```

Comment:
Thanks.  This file is typically auto-generated, must have been a hand-edit to add this.  Typically internally we do not recommend using the simCore.h, simQt.h, etc. files because they often bring in far more dependencies than one might want.  I've considered removing them.  The extra maintenance that this PR would require make me think that moreso.

Reply:
That is the route I eventually went to fix my local issues with it, but figured since they were still there, I would push a patch for them.